### ISSUE TITLE
Keyboard navigation needs improvement.

### ISSUE THREAD

**Issue by ephieo** [fac20/Week1-BDIM-DigitalAgency#58]
Keyboard - The site has the ability to tab through the navbar but it then jumps straight to the form.
People using a screen-reader wouldn’t be able to see the rest of the content on the page.

**Comment by RunGT** [comment]
Thank for this @ephieo. 

We have had pointers from @jackherizsmith and @oliverjam on this.
I believe it's the limitation of ChromeVox, a screen reader would have more functionality .

**Comment by oliverjam** [comment]
(I just posted something in Slack clarifying how to set up Chromevox to navigate non-focusable elements)
